node_modules/oae-authentication/emailTemplates/reset.html.jst

```diff
@@ -0,0 +1,194 @@
+<%
+    /**
+     * Render the email header
+     */
+    var renderHeader = function() {
+        var institutionalLogoURL = shared.ensureAbsoluteLink(util.ui.getHashedPath(skinVariables['institutional-logo-url'].replace(/\'/g, '')), baseUrl);
+        var greeting = util.i18n.translate('__MSG__ACTIVITY_EMAIL_GREETING__', {'displayName': util.html.encodeForHTML(user.displayName)});
+        greeting = shared.ensureAbsoluteLinks(greeting, baseUrl);
+
+        var str = '';
+        str += '<tr id="header" class="row">';
+        str += '    <td align="center">';
+        str += '        <table cellpadding="0" cellspacing="0">';
+        str += '            <tr id="logo">';
+        str += '                <td><img src="' + institutionalLogoURL + '" /></td>';
+        str += '            </tr>';
+        str += '            <tr id="greeting">';
+        str += '                <td><h1>' + greeting + '</h1></td>';
+        str += '            </tr>';
+        str += '        </table>';
+        str += '    </td>';
+        str += '</tr>';
+
+        print(str);
+    };
+%>
+
+<%
+    /**
+     * Render the email footer
+     */
+    var renderFooter = function() {
+        var copyrightSymbolI18n = util.i18n.translate('__MSG__COPYRIGHT_SYMBOL__');
+        var apereoFoundationI18n = util.i18n.translate('__MSG__APEREO_FOUNDATION__');
+        var url = baseUrl + '/?me';
+
+        var str = '';
+        str += '<tr id="footer" class="row">';
+        str +=   '<td align="center">';
+        str +=       util.i18n.translate('__MSG__ACTIVITY_EMAIL_CHANGE_PREFRENCES__', {'url': url}) + '<br />';
```

Comment:
Maybe just get rid of this line. It doesn't make much sense to remind a user he can change his email preferences in a "reset password" email